velox/substrait/VeloxToSubstraitType.h

```diff
@@ -28,12 +28,17 @@ class VeloxToSubstraitTypeConvertor {
   /// Convert Velox RowType to Substrait NamedStruct.
   const ::substrait::NamedStruct& toSubstraitNamedStruct(
       google::protobuf::Arena& arena,
-      const velox::RowTypePtr& rowType);
+      const velox::RowTypePtr& rowType,
+      std::vector<bool> nullableList);
 
   /// Convert Velox Type to Substrait Type.
   const ::substrait::Type& toSubstraitType(
```

Comment:
What does it mean for a type to be nullable? Velox types do not have this attribute.

Reply:
Please see: https://substrait.io/types/type_system/
![image](https://user-images.githubusercontent.com/52772496/176577195-2c31efa9-d51c-4c4d-9fb9-e2496b281714.png)
The example of I32  
```
message I32 {
    uint32 type_variation_reference = 1;
    Nullability nullability = 2;
  }
```

Reply:
@ZJie1 My reading of this document suggests that we should always set nullable to true as all Velox types allow null values.

Reply:
This make sense . Will change it.

Reply:
Thanks.